.github/dependabot.yml

```diff
@@ -0,0 +1,7 @@
+version: 2
+updates:
+- package-ecosystem: gitsubmodule
+  directory: "/"
+  schedule:
+    interval: monthly
+  open-pull-requests-limit: 10
```

Comment:
this could be 1 xP

Reply:
It's just copied from https://github.com/rust-vmm/vmm-sys-util/blob/main/.github/dependabot.yml

Reply:
yeah, its definitely not a blocker :)